### PR TITLE
Updates for ISD Generation

### PR DESCRIPTION
**Pull request by jlaura**
Not for merge yet.

### CODE REVIEW

**Comment by jlaura** [comment]
@thareUSGS @dpmayerUSGS Can you two checkout the proposed move of the io_gpf stuff into io_bae.  Let me know how breaking this change might be?  Nothing having to do with API, but definitely having to do with how imports are going on.

File: plio/io/io_bae.py [hunk 1]
```diff
@@ -1,88 +1,133 @@
+import json
```
Comment (thareUSGS) [inline plio/io/io_bae.py]: #This file contains method to read and write various BAE(tm) file formats
#specifically to support their photogrammtery software Socet SET and Socet GXP

Comment (jlaura) [reply plio/io/io_bae.py]: In reviewing another PR (https://github.com/USGS-Astrogeology/PDS-Pipelines/pull/7) I commented that I thought that we should do a short get together on python docos and packaging - I think this comment 100% belongs in the documentation for this module.

**Comment by dpmayerUSGS** [comment]
The move will definitely break import statements in https://github.com/USGS-Astrogeology/socet_gxp_dev/blob/master/SurfaceFit/surface_fit_pc_align.py,
but that code is still research-y so changing it isn't a big deal yet.

**Comment by jlaura** [comment]
Ill get a PR put in there as well.

File: plio/io/io_bae.py [hunk 2]
```diff
@@ -1,88 +1,139 @@
...
-        outGPF.write('{0} {1} {2}\n'.format(row['point_id'], row['stat'], row['known']))
-        outGPF.write('{0}         {1}         {2}\n'.format(row['lat_Y_North'], row['long_X_East'], row['ht']))
-        outGPF.write('{0} {1} {2}\n'.format(row['sigma0'], row['sigma1'], row['sigma2']))
-        outGPF.write('{0} {1} {2}\n\n'.format(row['res0'], row['res1'], row['res2']))
-
-    outGPF.close()
-    return
+import pandas as pd
+
+def socet_keywords_to_json(keywords, ell=None):
```
Comment (thareUSGS) [inline plio/io/io_bae.py]: call this socetset_*

**Comment by jlaura** [comment]
@thareUSGS This should be good to go.

**Comment by jlaura** [comment]
I had not realized that this PR was still open....assuming the CI is clean.  I am going to merge based on comments from @thareUSGS and @dpmayerUSGS.

**Pull request merged**